WalletWasabi/Helpers/Constants.cs

```diff
@@ -55,7 +55,7 @@ public static class Constants
 
 		public static readonly Version ClientVersion = new(1, 1, 13, 0);
 		public static readonly Version HwiVersion = new("2.0.2");
-		public static readonly Version BitcoinCoreVersion = new("0.21.1");
+		public static readonly string BitcoinCoreVersion = new("");
```

Comment:
This feels weird to me.

Reply:
Reverted in https://github.com/zkSNACKs/WalletWasabi/pull/6942/commits/af03334456ed50f28b622d47f0d15d0e8c3a428f